umake/ui/cli/__init__.py

```diff
@@ -157,6 +158,41 @@ def mangle_args_for_default_framework(args):
     return result_args
 
 
+def get_frameworks(args):
```

Comment:
I would call it "get_frameworks_list(args)"

or even better:
get_frameworks_list(installed_only=False)

And set installed_only based on args.<vars> outside.

also, please add a description :)